### DISCUSSION TITLE
Introspection bug/request:  Using SFDX, How to list  all polymorphic tables ie those that are retrievable but not present in schema:sobject:list? Example: "Activity"

### DISCUSSION THREAD

**Discussion by peternoges** [forcedotcom/cli#2372] [Ideas]
Hello.  

Observed behavior:
 `sfdx force:source:retrieve --json -m CustomObject:Activity` runs fine, but none of the commands below return "Activity":

- `sfdx force:schema:sobject:list -c standard --json > ActivityDoesntAppearHere.json;`
- `sfdx force:schema:sobject:list -c all --json > ActivityDoesntAppearHereEither.json;`
- `sfdx force:schema:sobject:list -c custom--json > ActivityDoesntAppearHereAsWell.json;`

Question & Suggestion:

Are there any existing force commands that can return "Activity" and any other polymorphic concepts? Having an introspection command that reveals all of them would be hugely helpful.

I know "`Activity`" is not a sObject,  but rather a polymorphic construct that can mean either "Task" or "Event" sObject,  therefore probably doesnt belong in the schema:sobject:list command,  however i think it deserves to belong in some introspective/describe command; afterall it can be CLI retrieved, orgbrowser retrieved, and source tracked. 

Can someone help make it describable? 

Suggestion:
Perhaps add a new command such as “force:schema:list:polymorphicObjects” 

And are there others like Activity that share the observed behavior above?

Thank you,

[a comment by an automated account is not shown]

**Comment by peternoges** [comment]
Is there a way for us to access information about [W-8793485] ?

**Comment by mshanemc** [comment]
we could if there was an API for them, but I don't see them.  Like you can 
`sf sobject describe --sobject Event` but not `sf sobject describe --sobject Activity`
